test/parallel/test-runner-output.mjs

```diff
@@ -98,6 +98,7 @@ const tests = [
   { name: 'test-runner/output/global-hooks-with-no-tests.js' },
   { name: 'test-runner/output/before-and-after-each-too-many-listeners.js' },
   { name: 'test-runner/output/before-and-after-each-with-timeout-too-many-listeners.js' },
+  { name: 'test-runner/output/force_exit.js', transform: specTransform },
```

Comment:
Can you add another test that force exit is used but some tests are failing and check that the exit code is correct